src/main/java/org/radarbase/appserver/service/protocol/GithubClient.java

```diff
@@ -0,0 +1,97 @@
+/*
```

Comment:
can you move to parent directory `service` since this is not specific to protocols?